bitnami/elasticsearch/templates/master/statefulset.yaml

```
@@ -83,9 +83,9 @@ spec:
       {{- if .Values.master.terminationGracePeriodSeconds }}
       terminationGracePeriodSeconds: {{ .Values.master.terminationGracePeriodSeconds }}
       {{- end }}
-      {{- if or (or .Values.initContainers .Values.master.initContainers) .Values.security.enabled .Values.sysctlImage.enabled (and .Values.volumePermissions.enabled .Values.master.persistence.enabled) }}
+      {{- if and .Values.enableDefaultInitContainers (or (or .Values.initContainers .Values.master.initContainers) .Values.security.enabled .Values.sysctlImage.enabled (and .Values.volumePermissions.enabled .Values.master.persistence.enabled)) }}
```

If I am not wrong, we should set the initContainers if users set any of these values:
* `initContainers`
* `master.initContainers`
* `enableDefaultInitContainers` and `security.enabled ` 
* `enableDefaultInitContainers` and `sysctlImage.enabled`

So the condition should be:
```suggestion
      {{- if or .Values.initContainers .Values.master.initContainers (and  .Values.enableDefaultInitContainers (or .Values.security.enabled .Values.sysctlImage.enabled)) }}
```